### DISCUSSION TITLE
iPerl Watermeter 5.0 Problems

### DISCUSSION THREAD

**Discussion by schnuller123** [jomjol/AI-on-the-edge-device#3434] [Q&A]
Hello
my watermeter reader works very well, but every 50s shows my watermeter
in the Display 5.0, and the the Reading at this moment makes error
i cant find informations what this means 
![Screenshot_20241213-195639_Chrome](https://github.com/user-attachments/assets/d4e19b9b-0c89-415c-8b0d-a278c580ed9c)

**Comment by caco3** [comment]
Thats not to hard to find out.
I guess you have this device, right?
https://www.xylem.com/de-de/products--services/metrology-equipment-for-utilities/meters/iperl-international-water-meter/documentation/

If you scroll down, there is a user manual:
https://www.xylem.com/siteassets/brand/sensus/resources/manual/sensus-iperl-installation-manual.pdf

And in there I found this:
![grafik](https://github.com/user-attachments/assets/5a5c2978-39d9-474b-b0ff-f0332789dcf0)

![grafik](https://github.com/user-attachments/assets/2f467c22-85db-408d-9cfa-36039ef5faa3)

So I guess it is just the firmware version and can be ignored.

---

Now, the tricky part:
AIOTED do not have functionality to detect multifunctional displays.
All you can do is to disallow neg. flow (which you already have), and a tight max rate.

**Reply by schnuller123** [reply]
thanks for your help, this doku i did not found
but there is a big problem for me
because i git a lot of errors
because fhe message comes always at a wrong time
most on flow detect moment.

there is no change to ignore this bad info?

**Reply by schnuller123** [reply]
the problem is
i got a lot of flows with error
after 30-60 minutes it got a new value and then its ok
so long to the next 5.0 display

**Reply by SybexX** [reply]
@schnuller123 take a look there https://github.com/jomjol/AI-on-the-edge-device/issues/3389#event-15485523912

**Reply by caco3** [reply]
@SybexX This IMO has nothing to do with the alignment!
@schnuller123 Do you see a pattern when the `5.0` gets shown? Is it periodic? According to the manual, it is part if the selftest. But there is no information if it is run periodically.
To improve the reading, set the interval to a higher rate, eg. 2 minutes, then you have a highter chance of valid readings.

**Reply by SybexX** [reply]
@caco3 Please read the content first and see what the user intends to do with it. Then you will probably have a different opinion.

**Reply by caco3** [reply]
Please hint, me, I seem to "stand on the tube" :smile:

**Comment by schnuller123** [comment]
hi
the version message and a displaytest comes every 60s
